lean/src/valid.lean

```diff
@@ -1455,8 +1457,8 @@ begin
 end
 
 theorem amc12a_2020_p22
-  (h₀ : fintype {n : ℕ | 5∣n ∧ nat.lcm (nat.factorial 5) n = 5 * nat.gcd (nat.factorial 10) n}) :
-finset.card {n : ℕ | 5∣n ∧ nat.lcm (nat.factorial 5) n = 5 * nat.gcd (nat.factorial 10) n}.to_finset = 1 :=
+  (h₀ : fintype {n : ℕ | 5 ∣ n ∧ nat.lcm (nat.factorial 5) n = 5 * nat.gcd (nat.factorial 10) n}) :
+  finset.card {n : ℕ | 5 ∣ n ∧ nat.lcm (nat.factorial 5) n = 5 * nat.gcd (nat.factorial 10) n}.to_finset = 1 :=
```

Comment:
you can rewrite this one the same way as `mathd_algebra_405`. Also, maybe we could use `!` for `nat.factorial` here?